bib/east.bib

```diff
@@ -6890,15 +6896,6 @@ @article{east:5099
   pages =        {39--87},
 }
 
-@Book{east:5100,
-  date       = {0710},
-  title      = {觀所緣緣論釋 // {Guan} suo yuan yuan lun shi},
-  language   = {zh-CN},
-  series     = {Taishō 1624; {Koryŏ} taejanggyŏng 625},
-  translator = {{Yi Jing}},
-  keywords   = {canonical},
-}
-
```

Comment:
Deleting this would leave https://east.ikga.oeaw.ac.at/buddh/ind/5/177/186/ without bibliographic support.  Is this a duplicate of some other entry?

Reply:
Yes, east:5100 is a duplicate of east:5029. Also the Taisho nr. given in east:5100 is wrong, it should be Taisho 1625.

Reply:
I've replaced the text-relation of east: 5100 to Dharmapāla's commentary on ālambanaparīkṣā/vṛtti with east: 5029. But there comes up another problem. https://east.ikga.oeaw.ac.at/buddh/ind/5/177/#1431. On the page, the author name is followed by 710, which is wrong, since 710 is the year of translation, not the year of composition. I think this problem would also come up with other pages for other classical Chinese translations after I supplied the author name.

Reply:
A reply is in https://github.com/ea-east/east-biblio/pull/18#issuecomment-621045022.